.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,24 @@
+name: .NET
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: windows-latest
+    timeout-minutes: 15
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.x
```

Comment:
I saw you fixed the new() issue that is only valid in .NET 5+. Did you try changing this to 5.0.x?

Reply:
No problem! I was curious to figure GitHub Actions out, since they're kind of new.

Yep, I tried that out, and it doesn't find the 3.1.x framework there for testing.

<img width="1115" alt="image" src="https://user-images.githubusercontent.com/16342951/127195404-c04d6880-9aa2-43c0-90ea-3e4a8d1b9372.png">